### ISSUE TITLE
Port Status

### ISSUE THREAD

**Issue by CDAGaming** [CDAGaming/Svarka#1]
Current Issues (Pinging @SantaGitHub)

1) For MCP Version, it would say MCP Unknown when it should say "MCP 9.40" like how it should be for the 1.12.1 port

**Comment by CDAGaming** [comment]
100 Errors with New MC Patches
Commiting Shortly

**Comment by SantaGitHub** [comment]
Помогу только завтра или в выходные. Еду в поезде)

**Comment by CDAGaming** [comment]
Ok, I'll add you as contributor, though one thing, push changes to the dev branch so when I merge it into master I can keep it clean

**Comment by CDAGaming** [comment]
Odd @SantaGitHub It seems on your repo, i also get the same error here:

Execution failed for task ':makeJarSvarka'.
> Could not execute build using Gradle installation 'C:\Users\tonyg\.gradle\wrapper\dists\gradle-2.13-bin\4xsgxlfjcxvrea7akf941nvc7\gradle-2.13'.

I'll organize the issues based on priority

**Comment by CDAGaming** [comment]
@SantaGitHub Updated Issue Info 👍

**Comment by CDAGaming** [comment]
@SantaGitHub Hope you will be available soon to assist on this

**Comment by djoveryde** [comment]
Where are the source patches located for updating?

**Comment by CDAGaming** [comment]
@djoveryde In src/main
For Generating patches for patches folder you'd run gradlew genPatches

**Comment by djoveryde** [comment]
Its acting as though its missing a package as its not importing net.minecraft.*

**Comment by CDAGaming** [comment]
Can you give the full error, and did it produce same output on Santa's or yours

Also remember to use the dev branch as that's where the latest changes are

**Comment by djoveryde** [comment]
Its throwing the same gradle installation error but for some reason its complaining about code that isnt even part of Svarka but minecraft

**Comment by djoveryde** [comment]
https://pastebin.com/X8kP1QiL - Attempt to build

**Comment by CDAGaming** [comment]
Yup those were the patch issues i was talking of earlier (Of course, they are different for me on 1.12.1)
@SantaGitHub Could you perhaps lend a hand here?

**Comment by djoveryde** [comment]
All I can see is that the patch checker isnt grabbing the core minecraft package by the look of it so its just giving up
